travis/test_server.py

```diff
@@ -136,10 +136,6 @@ def get_addons_path(travis_dependencies_dir, travis_build_dir, server_path):
     return addons_path
 
 
-def get_server_script(odoo_version):
-    return 'odoo-bin' if float(odoo_version) >= 10 else 'openerp-server'
```

Comment:
The idea is to accomplish it by the proposed file copy _instead_ of using this method, which introduced the problems described in #398 in the first place. HI doubt having two alternative implementations of the same thing is what we want.